### PR TITLE
Add SAP as known instance for the 30 Day Warranty and Praise Participants patterns

### PR DESCRIPTION
**Pull request by michael-basil**
Recently during one of our internal SAP InnerSource focused workstream meetings I inquired about guidance for making a pull request to add SAP as a known instance for existing patterns.  The conversation with @dellagustin surfaced an idea that a known instance for a pattern should have a basis for such a claim.

As such, I have captured the spirit of the value we are finding in one of my home group/departments as described in the following blog article:

* https://community.sap.com/t5/open-source-blogs/innersource-first-contribution-explored/ba-p/13644916

I linked the article to the known instance in the proposed modified files.

Please let me know how this sits and or if there are changes desired.

Warmly.
Michael Basil 🙏 🌿

### CODE REVIEW

**Comment by michael-basil** [comment]
I see that your PR process does a nice modeling of praise participants!

I have reviewed the contribution guidelines.

Additionally, I see that all the checks have passed.

I think the PR is in a good state to be reviewed at this point.

I am humbly awaiting your feedback!

**Review by spier — COMMENTED** [review]
Hi @michael-basil, thank you for sharing that SAP is using these patterns!

Besides sharing **that** you are using the patterns, it could be super helpful if you could also share a bit about **how** you are applying them.

As patterns have to be adapted to the specific context at the organization, it is a good illustration for other potential adopters to share more about the specifics of how this worked at SAP.

I took a look at the blog post you referenced. Here some questions that could lead to great illustrations of how applied the pattern:

- how does the praise participants approach relate to SAP's Appreciate program?
- how do you share the personalized thank you notes? written? digital?
- given SAP's size, where do you post such notes? i.e. what internal forums are helpful for saying thank you?
- for the 30 day warranty: which period did you apply? i.e. also 30 days, or something else, and why?
- are there any projects that don't need the 30 day warranty pattern? i.e. for what kind of projects is that pattern helpful, and for which ones not?

Looking forward to your feedback!

**Comment by michael-basil** [comment]
Hello @spier,

Thank you for the mindful feedback.

I have submitted a new version of the blog to the community.sap.com relevant community leader.  This has been reviewed and approved and is available at the link below.

* https://community.sap.com/t5/open-source-blogs/innersource-first-contribution-explored/ba-p/13644916

I attempted to channel a bit more breadth and depth with this revision after re-reviewing the 2 patterns in play here.

Does this version adequately address the questions and spirit of your prompts?

Also, I guess we are having a First Contribution experience?! 😁

Warmly.
Michael Basil 🙏 🌿

**Comment by spier** [comment]
Hi @michael-basil.

Yes, the examples that you added to the blog post are helpful. It is nice that you have multiple different projects that you can reference in those examples.

I was originally wondering if you want to add any of this information to these patterns itself? You could think of it like a teaser in the "Known Instances" section, highlighting how and why SAP is applying these patterns and motivating readers to click through to your blog post? Maybe just something to consider for future PRs, as is doesn't to stop us from merging this PR already.

Therefore I am already going ahead and merging this PR.

As an aside:
In you blog post there is one word missing in this bit "community maintainers are becoming more aware the". Should be "community maintainers are becoming more aware of the" I think?

File: patterns/2-structured/30-day-warranty.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ In addition it helps to provide clear [contribution guidelines](./base-documenta
 - This was tried and proven successful at PayPal.
 - GitHub internally uses this pattern with a modified warranty timeline of 6 weeks.
 - Microsoft recommends this pattern as a principle - teams set their own specific time target matching their needs and confidence.
+- SAP
+  - See: [InnerSource: First Contribution Explored](https://community.sap.com/t5/open-source-blogs/innersource-first-contribution-explored/ba-p/13644916)
```
Comment (spier) [inline patterns/2-structured/30-day-warranty.md]:
```suggestion
- SAP - see: [InnerSource: First Contribution Explored](https://community.sap.com/t5/open-source-blogs/innersource-first-contribution-explored/ba-p/13644916)
```

File: patterns/2-structured/praise-participants.md [hunk 2]
```diff
@@ -68,6 +68,8 @@ Overdoing it may feel insincere and mechanical and defeat your purpose in reachi
 ## Known Instances
 
 * Nike (multiple projects)
+* SAP
+  * See: [InnerSource: First Contribution Explored](https://community.sap.com/t5/open-source-blogs/innersource-first-contribution-explored/ba-p/13644916)
```
Comment (spier) [inline patterns/2-structured/praise-participants.md]:
```suggestion
* SAP - see: [InnerSource: First Contribution Explored](https://community.sap.com/t5/open-source-blogs/innersource-first-contribution-explored/ba-p/13644916)
```

**Review by spier — APPROVED** [review]

**Comment by spier** [comment]
Changes are now live. e.g. see https://patterns.innersourcecommons.org/p/30-day-warranty#known-instances

Nice job on your first contribution @michael-basil ! ❤️

**Comment by michael-basil** [comment]
Thanks @spier.  Much appreciated.

I was on the fence about putting the teaser or not.  I could be sold either way.  

I may push this question internally at SAP to @dellagustin to see what he thinks.

We have some ongoing dialog about exploring this pattern in further detail internally, so there may be a bit more teaser breadth and depth quality through this dialog.

I really enjoyed this First Contribution experience.  It is giving me some ideas internally that I am making a mental note and will percolate on.

Lastly, how does one become a "member" of the InnerSource Commons GitHub Organization?  And what is entailed should such a designation be granted?

**Comment by michael-basil** [comment]
Oh, thanks for the note on the typo ... I will correct that and do one more read through.  I think I found one more.

**Comment by dellagustin-sap** [comment]
Hi @michael-basil , if you can come up with a good teaser, that would be helpful for the readers of the pattern.
To add a bit, @michael-basil is a contributor of our "ISPO" (it is called something else, but functions in part* as an ISPO).
He's driving/advocating for InnerSource adoption on his home projects.
From a broader SAP context, we have an internal landing page for InnerSource, which contains a subpage with "internal patterns" and a link to the InnerSource patterns book. We are planning to eventually "fork" some of the ISC patterns and specialize them for SAP. The first candidate is the praise participants, as we can advocate for the usage tools and forums that are specific to SAP, e.g., the SAP Appreciate program and corresponding tool, mentioned by @michael-basil .

.* - Depends on your expectations of what an ISPO is, which to my best knowledge is currently not very well defined.

**Pull request merged**